.github/workflows/downgrade_versions.yml

```diff
@@ -0,0 +1,58 @@
+name: tree
```

Comment:
It's probably fine to group `entrypoint-tag` and `downgrade_versions` jobs in the same workflow `tree` but is it possible to spread one workflow among different files in GitHub? If it is possible and you decide to group the two jobs in one workflow, better rename `downgrade_versions` to `downgrade-version` for consistency with `entrypoint-tag`.

Reply:
This is just copy-paste mistake. I'd better use `downgrade_versions`.